Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "depub"
+version = "0.1.0"
+authors = ["Laurence Tratt <laurie@tratt.net>"]
+edition = "2018"
+readme = "README.md"
+license = "Apache-2.0 OR MIT"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
I usually nuke this line.

Reply:
Fixed in https://github.com/softdevteam/depub/pull/1/commits/f093ea559f556b89df89ba5df9e13ba757b1e76b.